hi-ml-histopathology/src/histopathology/datamodules/base_module.py

```diff
@@ -291,7 +291,7 @@ def _load_dataset(self, slides_dataset: SlidesDataset, stage: ModelKey) -> Datas
                     keys=slides_dataset.IMAGE_COLUMN,
                     reader=WSIReader,
                     backend="cuCIM",
-                    dtype=np.float32,
+                    dtype=np.uint8,
```

Comment:
This change could have quite a bit of impact (Running transforms on uint8 versus float ). Why was that necessary? What's the impact of that change alone? Tiling is probably lots faster when working with uint8.

Reply:
Yes! the first improvement I made was to load/tiles the slides in uint8 and later cast it to float (see deepmil.py line 207).
It reduced the training epoch time from 432 min -> 110min. 
I can also check how much of improvement this is with level 1